app/services/user-settings.js

```diff
@@ -0,0 +1,67 @@
+import EmberObject from '@ember/object';
+import Service from '@ember/service';
+import {getObject, setObject} from '../utils/storage';
+
+const SETTINGS_KEY = 'user_settings';
+const TOKENS_KEY = 'user_tokens';
+const PROPERTIES = ['githubTokenModalSeen'];
+
+// Use github_com instead of github.com for localstorage because of ember `set`
+// and `get` internals
+const userSettings = Service.extend({
+  init() {
+    this._super(...arguments);
+    this.tokens = EmberObject.create({
+      'github_com': null,
+    });
+    this.reload();
+  },
+
+  reload() {
+    this.reloadSettings();
+    this.reloadToken();
+  },
+
+  reloadSettings() {
+    const settings = getObject(SETTINGS_KEY) || {};
+
+    // Only set property defined in PROPERTIES
+    for(let property of PROPERTIES) {
```

Comment:
You've got some for loops with a space between the `for` and the opening parenthesis, and some without. Should be fixed with the .coafile, but I believe we usually go _with_ a space.

Reply:
Alright

Reply:
@bekicot Should've added style rules to .eslintrc like I said